shared/folders/files/render.native.js

```diff
@@ -91,7 +91,8 @@ const styleHeaderThemed = {
 }
 
 const styleTLFHeader = {
-  height: 64,
+  ...globalStyles.flexBoxColumn,
+  minHeight: 64,
```

Comment:
Is minHeight supported on Android now? I can't find a mention of it in the changelog for RC 0.30. While iOS support is mentioned in RC 0.29.